pkg/apis/maestro/v1alpha1/frameworkversion_types.go

```diff
@@ -88,6 +88,10 @@ type Parameter struct {
 	//Default is a default value if no paramter is provided by the instance
 	Default string `json:"default,omitempty"`
 
+	//Trigger identifies the plan that gets executed when this parameter changes in the Instance object.
+	//Default is `update` if present, or `deploy` if not present
```

Comment:
What is meant by default if present (i would assume present means I specify it explicitly)?